DSMRloggerAPI.ino

```diff
@@ -380,117 +384,145 @@ void setup()
   sprintf(cMsg, "Last reset reason: [%s]\r", ESP.getResetReason().c_str());
   DebugTln(cMsg);
 
-  telegramInterval = millis() + 5000;
-  noMeterWait      = millis() + 5000;
-  upTimeSeconds    = (millis() / 1000) + 50;
-  nextSecond       = millis() + 1000;
-
 #if defined( HAS_OLED_SSD1306 ) || defined( HAS_OLED_SH1106 )
     oled_Print_Msg(0, "<DSMRloggerAPI>", 0);
     oled_Print_Msg(1, "Startup complete", 0);
     oled_Print_Msg(2, "Wait for first", 0);
     oled_Print_Msg(3, "telegram .....", 500);
 #endif  // has_oled_ssd1306
-  
-} // setup()
 
 
+} // setup()
+
 //===========================================================================================
-void loop () 
+//===[ blink status led in ms ]===
+void blinkLEDms(uint32_t iDelay){
+  //blink the statusled, when time passed... #non-blocking blink
+  DECLARE_TIMER_MS(timerBlink, iDelay);
+  CHANGE_INTERVAL_MS(timerBlink, iDelay);
+  if (DUE(timerBlink))
+    blinkLEDnow();
+}
```

Comment:
Ongetwijfeltleuk zo’n knipperende led, maar de enige feedback die een logger zonder oled scherm heeft is de blauwe led. Die geeft aan of er een telegram binnen is gekomen.
Let wel: het lezen en verwerken van telegrammen is de core business van de logger. Nu knippert er een led zonder enige feedback over telegrammen die al-dan-niet binnen komen